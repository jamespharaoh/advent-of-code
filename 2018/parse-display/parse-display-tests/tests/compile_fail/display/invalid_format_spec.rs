use parse_display::Display;
#[derive(Display)]
#[display("{0:y}")]
struct TestStruct(u32);

fn main() {}
