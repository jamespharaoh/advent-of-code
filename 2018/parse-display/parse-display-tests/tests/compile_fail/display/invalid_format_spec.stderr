error: unknown format trait `y`
 --> $DIR/invalid_format_spec.rs:3:14
  |
3 | #[display("{0:y}")]
  |              ^
  |
  = note: the only appropriate formatting traits are:
          - ``, which uses the `Display` trait
          - `?`, which uses the `Debug` trait
          - `e`, which uses the `LowerExp` trait
          - `E`, which uses the `UpperExp` trait
          - `o`, which uses the `Octal` trait
          - `p`, which uses the `Pointer` trait
          - `b`, which uses the `Binary` trait
          - `x`, which uses the `LowerHex` trait
          - `X`, which uses the `UpperHex` trait
